src/view_curses.cc

```diff
@@ -322,8 +324,22 @@ view_curses::mvwattrline(WINDOW* window,
             int awidth = attr_range.length();
             nonstd::optional<char> graphic;
 
+            short graphic_inside_style = 0;
+
             if (iter->sa_type == &VC_GRAPHIC) {
                 graphic = iter->sa_value.get<int64_t>();
+                for (auto iter2 = sa.begin(); iter2 != sa.end(); ++iter2) {
+                    if ((iter2->sa_type != &VC_GRAPHIC) &&
```

Comment:
There are many different types of attributes, some for styling and some contain metadata, like the name of the file where a line is from.  So, it would not be enough to only check for non-VC_GRAPHIC attributes.

Reply:
Well, I am just checking for any attribute overlapping the area of the current VC_GRAPHIC element. The check on `iter2->sa_type` is meant to exclude the same attribute that's assigned to `iter`. I assumed, but correct me if I am wrong, that the only attribute crossing/overlapping (spatially) VC_GRAPHIC elements would be VC_STYLE elements for the coloured bars in the background. Feel free to correct me if there are other elements which might overlap I am unaware about.